apps/files_external/css/external.css

```
@@ -1,3 +1,4 @@
 .files-filestable tbody tr.externalErroredRow {
-	background-color: #F2DEDE;
+	/* TODO: As soon as firefox supports it: color-mix(in srgb, var(--color-error) 15%, var(--color-main-background)) */
+	background-color: #f002;
```

I'm really not fond of hex syntax for alpha values :confused: 
```suggestion
	background-color: rgba(255, 0, 0, 0.13);
```

---

:+1:  fixed